src/i18n/en-US/draftModelPlan/participantsAndProviders.ts

```diff
@@ -57,8 +57,8 @@ const participantsAndProviders = {
   },
   recruitParticipants: 'How will you recruit the participants?',
   recruitOptions: {
-    loi: 'LOI (Letter of interest)',
-    rfa: 'RFA (Request for applications)',
+    loi: 'LOI (Letter of intent)',
+    appCollectionTool: 'Use an application collection tool',
     recruitInfo:
       'CMMI writes, Office of General Council (OGC) reviews and approves',
     nofo:
```

Comment:
NOFO is missing the end parentheses  `)`

![image](https://user-images.githubusercontent.com/2349702/206541273-f12e0c0a-37af-4d62-91b9-2d20f2675bc8.png)